examples/macropad_mp3/happy.mp3.license

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: Copyright (c) 2021 Kattni Rembor for Adafruit Industries
```

Comment:
Should this be Copyright Bartlebeats instead? (presumably the same for all tracks)

Reply:
I was wondering the same thing. Wasn't sure about who to add here. Good call.